src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -19,7 +19,7 @@
                  @(_DebugSymbolsIntermediatePath);
                  $(NonExistentFile);
                  @(CustomAdditionalCompileOutputs)"
-          Returns=""
+          Returns="$(CscCommandLineInvocation)"
```

Comment:
:) One issue with that idea though is that folks will expect the GetCommandLineInvocation target to always work (regardless of project or global properties). Yet it won't, unless we copy the entire CoreCompile target so that it can invoke the Csc task itself.